### DISCUSSION TITLE
What information should be "recommended" in each template's docs?

### DISCUSSION THREAD

**Discussion by jcbhmr** [devcontainers-community/templates#8] [General]
> There is no agreement on what information should be included in the document at this time, so there are only minimal notes. Ideally, we should put the names of the maintainers, as in https://github.com/microsoft/vscode-dev-containers.

&mdash; @eitsupi https://github.com/devcontainers-community/templates/issues/2#issuecomment-1475156701

Here's some ideas of stuff that might be good to have:
- If it's transferred from somewhere else, include some kind of link, commit stamp, or archive.org or something to the original
- A list of usernames that a user can @mention to contact the maintainers/authors/relevant people
- What the template lets you do (what are the tools inside it?)

> **FUTURE IDEA** ~~Here's some other ideas that are more like "wrapper things" that would be good to have the same on all template docs. Sorta like a footer/header on a website:~~ These ideas kinda need a static site generator to do. Don't worry about them now.
> - ~~Link to create a new issue about a bug/crash~~ (more so for things like features, not templates)
> - Link to ask a question/help in Discussions Q&A
> - Link to suggest or add a new template

**Comment by jcbhmr** [comment]
Here's an example from https://github.com/devcontainers-community/features/tree/7efa43ce70e3ab498357830e9ba1a16df4f5c538/src/deno -- yes, it's a _feature_ not a template, but I think it's a pretty good example of what I like in documentation: _relevant information_ for users 😉

![image](https://user-images.githubusercontent.com/61068799/226230126-78623774-e5e2-4595-8884-c4cd38e9f730.png)

**Reply by eitsupi** [reply]
I think it is great to describe the general settings!.

We can also just add a comment in the jsonc (like devcontainers/templates#125 for example)

**Comment by jcbhmr** [comment]
Also as a reminder (this is more relevant for features) but when it crashes, the CLI will pop up saying something like:

```
Visit {documentationURL from devcontainer-*.json} for debugging help
```

So maybe it'd be good to have a sort of "need help? do this/create a Discussion" type thing in there too?
